CodeExecutor/AppDomainWorker.cs

```diff
@@ -91,6 +93,18 @@ public AssemblyMetaData GetAssemblyMetadata(string assemblyPath)
                 {
                     // if an exception occurs when loading custom attributes, just ignore
                 }
+
+                try
+                {
+                    data[AssemblyMetaData.ReferencedAssembliesKey] = assembly.GetReferencedAssemblies()
+                        .OrderBy(assName => assName.Name)
+                        .Aggregate(new StringBuilder(), (output, assName) => output.AppendLine(assName.FullName))
```

Comment:
looks good, but what about `string.Join`? That's a bit more clear IMO.

Reply:
Solved